script/Propose.s.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {Script} from "forge-std/Script.sol";
+import {ICompoundTimelock} from
+  "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+
+import {PooltogetherGovernor} from "src/PooltogetherGovernor.sol";
+import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
+
+contract Propose is Script {
```

Comment:
Similar to previous PR, let's update the README to include instructions for using and testing this script

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/7/commits/e361ba75debff3955ebd0f3ab5c0adac08031ac6